addons/calendar/models/calendar_event.py

```diff
@@ -211,6 +222,7 @@ def _compute_is_highlighted(self):
          ('confidential', 'Only internal users')],
         'Privacy', default='public', required=True)
     location = fields.Char('Location', tracking=True, help="Location of Event")
+    videocall_location = fields.Char('Join Video Call', default=_default_videocall_location, readonly=True)
```

Comment:
We could remove the readonly True, in case people already have a jitsi room for a meeating.